### PR TITLE
Input buttons integration

### PR DESCRIPTION
**Pull request by jfontana94**

### CODE REVIEW

File: de [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+[32m+[m[32m        GPIO.setmode(GPIO.BCM)[m[41m [m
+[32m+[m[32m        GPIO.setwarnings(False)[m
+[32m+[m
+[32m+[m[32m        GPIO.setup(self.btn_1, GPIO.IN, GPIO.PUD_UP)[m
+[32m+[m[32m        GPIO.setup(self.btn_2, GPIO.IN, GPIO.PUD_UP)[m
+[32m+[m[32m        GPIO.setup(self.btn_3, GPIO.IN, GPIO.PUD_UP)[m
+[32m+[m[32m        GPIO.setup(self.btn_4, GPIO.IN, GPIO.PUD_UP)[m
+[32m+[m[32m        GPIO.setup(self.btn_5, GPIO.IN, GPIO.PUD_UP)[m
+[32m+[m[32m        GPIO.setup(self.btn_6, GPIO.IN, GPIO.PUD_UP)[m
+\ No newline at end of file[m
```
Comment (EmaSuriano) [inline de]: Is this file is necessary for the project? I mean, it seems like a result of a build.

File: src/config/config.json [hunk 2]
```diff
@@ -1,4 +1,4 @@
 {
-    "input": "console",
+    "input": "buttons",
```
Comment (EmaSuriano) [inline src/config/config.json]: This is sooo cool!

File: src/input/buttons.py [hunk 3]
```diff
@@ -1,9 +1,62 @@
...
+    btn_3 = 16
+    btn_4 = 26
+    btn_5 = 19
+    btn_6 = 13
+
+    _MOVE_RIGHT = "right"
+    _MOVE_LEFT = "left"
+    _ENTER = "enter"
+    _BACK = "back"
+    _ENTER = "enter"
```
Comment (santicanti) [inline src/input/buttons.py]: Ya está declarado arriba

File: src/main.py [hunk 4]
```diff
@@ -18,10 +18,11 @@
 
 while letter != "bye":
     previous_state = state.get_state()
-    #letter = my_input.get_input()
-    state_object.process_input(my_input.get_input())
+    letter = my_input.get_input()
```
Comment (santicanti) [inline src/main.py]: Nombre de la variable -> input

**Review by santicanti — COMMENTED** [review]
Agregar el import de buttons.py en el condicional del processor helper

**Review by santicanti — APPROVED** [review]

**Pull request merged**